locale/en/docs/es6.md

```diff
@@ -55,12 +55,26 @@ All ES6 features are split into three groups for **shipping**, **staged**, and *
 
 * [Arrow Functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
 
+* [new.target](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/new.target)
+
+* [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe)
```

Comment:
Object.observe is not in ES6 (there is no standard for it so far) and Node 4 contains it as well. I don't think it belongs on this list.